binding.go

```diff
@@ -28,23 +31,51 @@ func RegisterBinder(contentType string, fn BinderFunc) {
 	binders[contentType] = fn
 }
 
-func init() {
-	schemaDecoder = schema.NewDecoder()
-	schemaDecoder.IgnoreUnknownKeys(true)
-	schemaDecoder.ZeroEmpty(true)
+// RegisterTimeLayout allows to add custom time layouts that
+// the binder will be able to use for decoding.
+func RegisterTimeLayout(layout string) {
+	timeLayouts = append(timeLayouts, layout)
+}
 
-	// register the types in the nulls package with the decoder
-	nulls.RegisterWithSchema(func(i interface{}, fn func(s string) reflect.Value) {
-		schemaDecoder.RegisterConverter(i, fn)
+// RegisterBinderTypeDecoder allows to define custom type decoders.
+func RegisterBinderTypeDecoder(fn formam.DecodeCustomTypeFunc, types []interface{}, fields []interface{}) {
+	decoder.RegisterCustomType(fn, types, fields)
+}
+
+func init() {
+	decoder = formam.NewDecoder(&formam.DecoderOptions{
+		TagName:           "schema",
```

Comment:
Let's set the tag to `form` instead. See this ticket https://github.com/gobuffalo/tags/issues/26